adds guidelines for what we are generally looking for (and what we ar… (#1752)
rust-lang/this-week-in-rust: README.md

## PRs for next issue are now being accepted

To propose content for inclusion in the next newsletter (found in the `drafts/`
folder), create a new [Pull Request][pr] updating the relevant section in the 
draft.

Alternately, tweet us [@thisweekinrust](https://twitter.com/thisweekinrust).

### What do we look for when considering whether to include something in This Week in Rust?

This Week in Rust is intended to highlight the incredible work of the Rust Community. 

What we are generally looking for includes:

* how-to intros (and advanced deep dives) into Rust concepts and areas

> * Rust walkthroughs

Reviewer: Lately, I've been excluding basic walkthroughs of Rust concepts already covered by the Rustlings course, Book, or Rust by Example (ie, "here's what a trait looks like" or "Hello, Rust!!!"). Should we maybe mention something like that in the "what we aren't looking for"?

Reviewer 2: I'm not sure if that's a good idea.
People have different styles to present, and people have different preferences how to consume.
I wouldn't want to see this excluded.

Author: I'd like to leave it in there - but maybe as editors we can focus on including things that may explain these concepts in different ways that what we have seen before?

Author: Actually - I think I know how to word this - adding it in.

Author: Done! Went with the wording "Rust walkthroughs that explain concepts in different ways than well known resources like [the Rust book](https://doc.rust-lang.org/stable/book/), [Rustlings](https://github.com/rust-lang/rustlings), and [Rust by Example](https://doc.rust-lang.org/stable/rust-by-example/)"

> * updates on tooling when in long form or framed as a tutorial (see what we are not looking for for more details)

Reviewer 3: ```suggestion
* updates on tooling when in long form or framed as a tutorial (for more details, see what we are not looking for below)
```

* Rust-related podcast episodes
* Rust-related screenshots and videos
* Rust meetup recordings
* Rust meetup announcements
* Presenter slide decks on Rust

> * Observations and throughts on Rust and the Rust community

Reviewer 2: ```suggestion
* Observations and thoughts on Rust and the Rust community
```

* Calls for participation in Rust open source projeccts
* Rust job announcements
* and more!

What we are generally NOT looking for includes:

* Anything that violates the [Rust Community Code of Conduct](https://www.rust-lang.org/policies/code-of-conduct)

> * Rants or anything degrading to any part or member of the Community. Rather than submitting an article about what is wrong with something, we would much rather you write something that explains how you made it better.

Reviewer 4: I feel this should be worded to not sound like gatekeeping.

```suggestion
* Rants or anything degrading to any part or member of the Community. Rather than submitting an article about what is wrong with something, we would much rather you write something that explains how you'd make it better.
```

Author: That is quite reasonable!

* Duplicates of recent posts (even with the wording changed slightly)
* Links to crates or GitHub repos without some sort of context. We would much rather you submit a blog post introducing your project and how a Rust user might use it (and what it would help them do) and/or what you learned about Rust in the process of writing the project.

> * Anything behind a paywall

Reviewer: Should we maybe explicitly state that this includes Medium's paid article mechanism?

Author: Good idea, I'll add that it :)
